doc/classes/EditorPlugin.xml

```diff
@@ -184,6 +185,7 @@
 			<return type="void">
 			</return>
 			<description>
+				This method is a called automatically when the [EditorPlugin] is enabled through the Project->Project Settings->Plugins interface
```

Comment:
`Called by Godot when the user enables the [EditorPlugin] in the Plugin tab of the project settings window`

Reply:
Can you check the new iteration, in this one I just copy pasted the description from forward_spatial_gui_input. Now I rewrote both.